New package: envision-3.2.0

lkroll opened the pull request:
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

This PR adds a package for Envision (https://gitlab.com/gabmus/envision) which is otherwise only available as an Appimage. This native package has a couple of benefits like better desktop integration (correct scaling) and working "open folder" buttons which won't be fixed in the Appimage (see https://gitlab.com/gabmus/envision/-/issues/96).

The libclang patch is necessary for the included dependency checker to work in the current release, but is already merged and will therefore not be necessary in the next tagged release, see [MR](https://gitlab.com/gabmus/envision/-/merge_requests/140).

lkroll committed:
New package: envision-3.2.0

Bnyro commented on srcpkgs/envision/template:
```diff
@@ -0,0 +1,18 @@
+version=3.2.0
+revision=1
+build_style=meson
+hostmakedepends="rust cargo git pkg-config gettext desktop-file-utils"
```
Why is `git` required at build time?

Usually that means that the app is trying to parse some information of its git repository, but in our case the `tar.gz` distfile doesn't contain the git repo information anyways, so it can/should be patched out. Not sure if that's also the case here though

lkroll replied:
From the meson.build file I can see that git is used to check if the build is a tagged release:

```
opt_profile = get_option('profile')

# if a profile isn't specified infer from git
if opt_profile == 'default'
  # are we building a tagged version?
  if run_command('git', 'describe', '--tags', '--exact-match').returncode() != 0
```

I can probably define a profile and skip this step.

lkroll commented:
I've rebased my branch and added configure_args + `cargo` and `rust` to hostmakedepends.
`libgit2-1.8-devel` is also required now because `git` is not present.
